inst/templates/ad_adpc.R

```diff
@@ -0,0 +1,513 @@
+# Name: ADPC
+#
+# Label: Pharmacokinetics Concentrations Analysis Dataset
+#
+# Description: Based on simulated data, create ADPC analysis dataset
+#
+# Input: pc, ex,  adsl
+library(admiral)
+library(dplyr)
+library(lubridate)
+library(stringr)
+
+library(admiral.test) # Contains example datasets from the CDISC pilot project or simulated
+
+# ---- Load source datasets ----
+
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions
+# as needed and assign to the variables below.
+# For illustration purposes read in admiral test data
+
+
+# Load PC, EX, VS and ADSL
+data("admiral_pc")
+data("admiral_ex")
+data("admiral_vs")
+
+data("admiral_adsl")
+
+adsl <- admiral_adsl
+
+# When SAS datasets are imported into R using haven::read_sas(), missing
+# character values from SAS appear as "" characters in R, instead of appearing
+# as NA values. Further details can be obtained via the following link:
+# https://pharmaverse.github.io/admiral/articles/admiral.html#handling-of-missing-values
+
+# Load EX
+
+ex <- convert_blanks_to_na(admiral_ex)
+
+# Load PC and fix format issue for PCDTC
+
+pc <- convert_blanks_to_na(admiral_pc) %>%
+  rename(PCDTM = PCDTC) %>%
+  mutate(PCDTC = as.character.Date(PCDTM, format = "%Y-%m-%dT%H:%M:%S")) %>%
+  select(-PCDTM)
```

Comment:
Should this format issue be fixed at the `admiral.test` level?

Reply:
Yes. Removed this formatting code.  It looks like the issue is resolved.  If there is still an issue, it should be fixed at `admiral.test` level.